src/components/Definition/Definition.module.scss

```diff
@@ -1,21 +1,35 @@
 @import "../../styles/variables";
 
+@keyframes fadeIn {
+  0% {
+    opacity: 0;
+  }
+  100% {
+    opacity: 1;
+  }
+}
+
 .definition_container {
+  opacity: 0;
+
+  animation: 1s ease-out 0s 1 fadeIn forwards;
+  animation-delay: 3s;
+
+  position: absolute;
```

Comment:
Why position absolute?

Reply:
I feel like these are questions, not requests for changes. Setting the position of these allows them to overlap. If they are static they will be arranged one after the other. If one is static, and the other relative (then relatively placed over the other) that isn't as responsive or stable (or my intention for that matter). To set it absolutely is telling it to be placed in the center of the div, even though there is another div overlapping it. Does that make sense?

Reply:
I know why you're using it, but _why_ are you using it? If you do `display: none` on the flash word itself, the definition won't be next to it, but on top of it.

I did actually have a change request which i forgot to mention -- could you reduce the animation delay 0.5 to 1 second? Additionally, (and more importantly), the definition is on top of the flash word after I go back and forth. Shown below:

![image](https://user-images.githubusercontent.com/75239264/130803979-450e5da8-47f9-4066-ac5e-ba64b5bcbd80.png)

Reply:
Oh, the visibility isn't controlled by the display property, it's controlled by opacity. So they have to overlap and technically both are "visible" they just alternate opacity.

I'll check the overlap on page return and fix the transition.